src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const thead = table.querySelector('thead');
+const tbody = table.querySelector('tbody');
+const tfoot = table.querySelector('tfoot');
+
+function changeColumns(...elements) {
+  elements.forEach((element) => {
+    const rows = [...element.querySelectorAll('tr')];
+
+    rows.forEach((row) => {
+      const children = row.children;
+      const lastChildren = children[children.length - 1];
+      const newCell = document.createElement(`${row.cells[0].tagName}`);
+
+      newCell.textContent = row.cells[1].textContent;
+      row.insertBefore(newCell, lastChildren);
+    });
+  });
+}
+
+changeColumns(thead, tbody, tfoot);
```

Comment:
You can simplify it like this
```suggestion
const table = document.querySelector('table');

table.rows.forEach((row) => {
  const cloneColumn = row.cells[1].cloneNode(true);

  row.lastElementChild.before(cloneColumn);
})
```